lib/OpenQA/WebAPI/Controller/API/V1/Mm.pm

```diff
@@ -52,7 +52,7 @@ JSON block with the list.
 # IMHO it should be replaced with get_children and removed
 sub get_children_status {
     my ($self) = @_;
-    my $status = $self->stash('status');
+    my $status = $self->stash('state');
```

Comment:
could you rename the variable name as well to be consistent? And how about the method name and hence the routing target?

Reply:
Once i have all the other incompatibilities fixed.